apps/users/templates/users/pwreset_request.html

```diff
@@ -11,6 +11,16 @@
     <form method="post" class="listing-footer">
       {{ csrf() }}
       <fieldset>
+        {% if request.GET.migrated %}
```

Comment:
I know that Django magic allows for `request.GET.migrated` but would `request.GET.get('migrated')` be ok?